modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/client/blockstream/BlockStreamContext.kt

```diff
@@ -0,0 +1,12 @@
+package jp.co.soramitsu.iroha2.client.blockstream
+
+import io.ktor.client.HttpClient
+import java.net.URL
+
+data class BlockStreamContext(
+    val apiUrl: URL,
+    val client: HttpClient,
+    val from: Long = 1,
+    val storage: BlockStreamStorage,
```

Comment:
this should be a collection of ones

Reply:
what to return from `fun subscribe()` in this case?